modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/validation/InvalidExtensionConfigTestCase.java

```diff
@@ -69,20 +69,20 @@ public void petStoreMissingRequiredParameterInsidePojo() throws Exception {
   }
 
   @Test
+  // TODO MULE-19350 Update this test
   public void operationWithExpressionConfigReference() throws Exception {
-    expectedException.expect(ConfigurationException.class);
+    expectedException.expect(InitialisationException.class);
```

Comment:
doesn't this break compatibility?

Reply:
no, this is becuasoe of a change in the test, not from change in the code

Reply:
No. If an invalid extension definition is provided, the current behavior is to throw `ConfigurationException`. This assertion means  that either behavior changed and a different exception is been thrown, or that this test cases is now failing for a different reason, which means the original scenario is no longer tested